### PR TITLE
strip whitespace via .Rproj; remove raw.githubusercontent

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**

### CODE REVIEW

File: R/addGitHubActions.R [hunk 1]
```diff
@@ -22,14 +22,17 @@
...
-    use_github_action(NULL, # name is not used when a url is passed
-                      "https://raw.githubusercontent.com/pik-piam/lucode2/master/.github/workflows/lucode2-check.yaml")
+  if (file.exists("DESCRIPTION") && desc("DESCRIPTION")$get("Package") == "lucode2") {
+    question <- "Replace inst/extdata/lucode2-check.yaml with .github/workflows/lucode2-check.yaml?"
+    if (md5sum("./.github/workflows/lucode2-check.yaml") != md5sum("./inst/extdata/lucode2-check.yaml") &&
+        (!interactive() || !askYesNo(question))) {
+      stop("inst/extdata/lucode2-check.yaml != .github/workflows/lucode2-check.yaml")
+    }
+    file.copy("./.github/workflows/lucode2-check.yaml", "./inst/extdata/", overwrite = TRUE)
+  } else {
```
Comment (tscheypidi) [inline R/addGitHubActions.R:32]: would it make sense to also check here whether a DESCRIPTION file exists in order to make sure that the function is applied to a package folder?

File: R/addGitHubActions.R [hunk 2]
```diff
@@ -22,14 +22,17 @@
...
+    actionPath <- system.file("extdata", "lucode2-check.yaml", package = "lucode2")
+    githubAction <- sub("autoupdate_schedule: weekly", "autoupdate_schedule: quarterly", readLines(actionPath))
```
Comment (tscheypidi) [inline R/addGitHubActions.R:34]: what happens here? why isn't that set in the file directly for all packages?

File: R/buildLibrary.R [hunk 3]
```diff
@@ -137,10 +139,16 @@ buildLibrary <- function(lib = ".", cran = TRUE, updateType = NULL, gitpush = FA
     message("Could not add GitHub Actions:", error)
   })
 
-  if (!file.exists("DESCRIPTION") || desc("DESCRIPTION")$get("Package") != "lucode2") {
-    fileUrl <- "https://raw.githubusercontent.com/pik-piam/lucode2/master/.pre-commit-config.yaml"
-    urlConnection <- local_connection(url(fileUrl))
-    preCommitConfig <- sub("autoupdate_schedule: weekly", "autoupdate_schedule: quarterly", readLines(urlConnection))
+  if (file.exists("DESCRIPTION") && desc("DESCRIPTION")$get("Package") == "lucode2") {
```
Comment (tscheypidi) [inline R/buildLibrary.R:142]: the following lines are more or less a code duplication of  addGithubActions, just applied on a different file, right? Would it perhaps make sense to merge both to also make sure that bugfixes are allways applied to both?

File: R/addGitHubActions.R (hunk 1 above)
Comment (pfuehrlich-pik) [reply R/addGitHubActions.R:32]: That seems like a good idea, right at the start of buildLibrary, right?

**Review by tscheypidi — APPROVED** [review]
looks good

**Pull request merged**